requirements/tests.txt

```diff
@@ -20,6 +20,7 @@ supervisor; python_version < '3'
 kubernetes<4.0
 psutil
 pyvmomi
+vSphere-Automation-SDK
```

Comment:
This should instead be added to `requirements/static/*.in` as we talked about, likely where `pyvmomi` is also present, and then the requirements need to be recompiled, `pre-commit run -av`

Reply:
Yep, I'll get that going...